bal/Makefile

```diff
@@ -4,3 +4,4 @@
 
 obj-m					+= bal.o
 ccflags-y := -I$(src)/include
+EXTRA_CFLAGS += -Wno-error=date-time
```

Comment:
This laine makes that usage of the __TIME__ macro returns warning in commpilation istead error.
The __TIME__ was used for debugging, to be sure which BAL module was inserted. The line will be removed from the Makefile.